### DISCUSSION TITLE
[Feature]: Manage logged in devices and log them out remotely

### DISCUSSION THREAD

**Discussion by JVT038** [immich-app/immich#1663] [Feature Request]
### Feature detail

An overview of all devices signed in with the current user account, and the ability to revoke their access, so the devices / browsers have to sign in again.
This is useful if a device is lost or if someone temporarily signed in and forgot to sign out.

Platforms: For web, mobile and there are probably some server-side changes needed for this.

### Platform

Web

**Comment by jrasm91** [comment]
Implemented in #2329.
